### PR TITLE
fix: When typing password, view password is disabled 

### PR DESCRIPTION
**Pull request by ahmdshrif**
### Fixed Issues

$ https://github.com/Expensify/App/issues/7597
### Tests

1. Open app
2. Navigate to Settings>Security>Change password
3.  Start typing password and click on Show password

- [x] Verify that no errors appear in the JS console

### QA Steps

- [x] Verify that no errors appear in the JS console

### Tested On

- [x] Web
- [x] Mobile Web
- [x] Desktop
- [x] iOS
- [x] Android

#### Web

https://user-images.githubusercontent.com/21364901/152928962-b779d427-0ef9-47db-91d9-0febd4b5c0cc.mov

#### Mobile Web

https://user-images.githubusercontent.com/21364901/152928951-701782e5-d56e-4a31-b380-7a276030e895.mov

#### Desktop

https://user-images.githubusercontent.com/21364901/152928086-109ae8fa-b750-4a28-97b9-f883a7f72f14.mov

#### iOS

https://user-images.githubusercontent.com/21364901/152927614-99782521-6f73-4ea0-90c2-b321af09d182.mov

#### Android

https://user-images.githubusercontent.com/21364901/152747755-a79c62b9-dca0-4f7e-9b3a-45c88c9b3893.mov

### CODE REVIEW

File: src/pages/settings/PasswordPage.js [hunk 1]
```diff
@@ -156,7 +156,14 @@ class PasswordPage extends Component {
                         onBackButtonPress={() => Navigation.navigate(ROUTES.SETTINGS_SECURITY)}
                         onCloseButtonPress={() => Navigation.dismissModal(true)}
                     />
-                    <ScrollView style={styles.flex1} contentContainerStyle={styles.p5}>
+                    <ScrollView
+                        style={styles.flex1}
+                        contentContainerStyle={styles.p5}
+
+                        // to allow children of the scroll view to catch taps. when keyboard is opened.
+                        // eslint-disable-next-line react/jsx-props-no-multi-spaces
```
Comment (parasharrajat) [inline src/pages/settings/PasswordPage.js:164]: We know what this prop does. Comment should reflect why this is used.

**Review by parasharrajat — CHANGES_REQUESTED** [review]

File: src/pages/settings/PasswordPage.js (hunk 1 above)
Comment (ahmdshrif) [reply src/pages/settings/PasswordPage.js:164]: ok, I think we can change it to 
// to allow the user to press the show password icon when keyboard is open.

Comment (parasharrajat) [reply src/pages/settings/PasswordPage.js:164]: Allow the user to click show password while password input is focused.

Comment (ahmdshrif) [reply src/pages/settings/PasswordPage.js]: ok done

**Review by parasharrajat — APPROVED** [review]
LGTM 
cc: @AndrewGable 

:ribbon: :eyes: :ribbon:  C+ reviewed

**Review by AndrewGable — APPROVED** [review]

**Pull request merged**